File: coupons-backend-php/api/domain/Category.php
<?php
class Category {
    public int $id_category;
    public string $name;
    public bool $is_enabled;

    public function __construct(int $id_category, string $name, bool $is_enabled) {
        $this->id_category = $id_category;
        $this->name = $name;
        $this->is_enabled = $is_enabled;
    }
}
?>

File: CouponsClientBackendC#/CouponsClient.BC/Models/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CouponsClient.BC.Models
{
    public class Category
    {
        public int id_category { get; set; }
        public string name { get; set; }
        public bool is_enabled { get; set; }
    }

}
